client/src/components/ReviewForm.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react"
+
+const ReviewForm = ({ postReview }) => {
+  const [newReview, setNewReview] = useState({
+    comment: "",
+    rating: "",
+  })
+
+  const handleInputChange = (event) => {
+    setNewReview({
+      ...newReview,
+      [event.currentTarget.name]: event.currentTarget.value,
+    })
+  }
+
+  const handleSubmit = (event) => {
+    event.preventDefault()
+    postReview(newReview)
+    clearForm()
```

Comment:
Looks like this is always going to clear the form on submission even if the post request returns an error.  Not something you necessarily need to change but should be aware of.